backend/mysql_utils.js

```diff
@@ -22,6 +22,27 @@ exports.createUser = function (email, pass, mysqlConnPool, callback) {
     });
 }
 
+
+exports.doesUserExist = function (email, mysqlConnPool, callback) {
+    const sql = `SELECT * from users where email_id='${email}'`;
+    mysqlConnPool.getConnection(function (err, conn) {
+        if (!err) {
+            conn.query(sql, function (err, result) {
+                conn.release();
+                if (!err) {
+                    // returning result.length as a boolean value
+                    return callback(null, result.length);
```

Comment:
Since the return value should be a Boolean, I think we should convert the length into a boolean before returning it.
```suggestion
                    return callback(null, !!result.length);
```